js/packages/openinference-instrumentation-langchain/test/langchain.test.ts

```diff
@@ -9,15 +9,35 @@ import { ChatPromptTemplate } from "@langchain/core/prompts";
 import { MemoryVectorStore } from "langchain/vectorstores/memory";
 import { ChatOpenAI, OpenAIEmbeddings } from "@langchain/openai";
 import { RecursiveCharacterTextSplitter } from "langchain/text_splitter";
+import { OTLPTraceExporter } from "@opentelemetry/exporter-trace-otlp-proto";
 import { loadQAStuffChain, RetrievalQAChain } from "langchain/chains";
 import "dotenv/config";
 import {
   OpenInferenceSpanKind,
+  RetrievalAttributePostfixes,
+  SemanticAttributePrefixes,
   SemanticConventions,
 } from "@arizeai/openinference-semantic-conventions";
 import { LangChainTracer } from "../src/tracer";
 import { trace } from "@opentelemetry/api";
-jest.useFakeTimers();
+// jest.useFakeTimers();
```

Comment:
intentional?

Reply:
yeah i have to uncomment it to send to phoenix, fixed it for the pr now